app/pods/publications/overview/all/controller.js

```diff
@@ -0,0 +1,115 @@
+import Controller from '@ember/controller';
+import { action, set } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import tableColumns from 'frontend-kaleidos/config/publications/overview-table-columns';
+
+const defaultColumns = [
+  'publicationNumber',
+  'shortTitle',
+  'publicationDueDate',
+  'status'
+]
+
+export default class PublicationsOverviewAllController extends Controller {
+  queryParams = {
+    page: {
+      type: 'number',
+    },
+    size: {
+      type: 'number',
+    },
+    sort: {
+      type: 'string',
+    },
+  };
+
+  page = 0;
+  size = 10;
+  sort = '-created';
+
+  @tracked columnsDisplayConfig;
+  tableColumns = tableColumns;
+
+  @tracked isLoadingModel = false;
+  @tracked isColumnsDisplayConfigPanelShown = false;
+
+  constructor() {
+    super(...arguments);
+
+    this.initColumnsDisplayConfig();
+  }
+
+  initColumnsDisplayConfig() {
+    let columnsDisplayConfig = this.loadColumnsDisplayConfig();
+    if (!columnsDisplayConfig) {
+      columnsDisplayConfig = this.getDefaultColumnsDisplayConfig();
+    }
+    this.columnsDisplayConfig = columnsDisplayConfig;
+  }
+
+  @action
+  changeColumnsDisplayConfig(config) {
+    this.columnsDisplayConfig = config;
+    this.saveColumnsDisplayConfig(this.columnsDisplayConfig);
+  }
+
+  @action
+  resetColumnsDisplayConfig() {
+    this.columnsDisplayConfig = this.getDefaultColumnsDisplayConfig();
+    this.isColumnsDisplayConfigPanelShown = false;
```

Comment:
1. Is dit best: panel sluiten bij reset?
2. Ik zou graag nog toevoegen dat het panel ook sluiten wanneer ernaast geklikt wordt.

Reply:
1. Te bekijken met de gebruikers wat zij willen. Hoeft deze PR niet te blokkeren. Kan achteraf nog makkelijk toegevoegd worden.
2. Hiervoor zal ik een nieuwe story maken zodat die deze story niet blokkeert.